src/StreamJsonRpc.Tests/JsonMessageFormatterTests.cs

```diff
@@ -112,6 +113,24 @@ public void JTokenParserHonorsSettingsOnSerializer()
         Assert.Equal("2019-01-29T03:37:28.4433841Z", value);
     }
 
+    [Fact]
+    public async Task MultiplexingStream()
```

Comment:
Just out of curiosity what are we testing with this? The Assert methods seem to me to be only testing the setting of formatter.MultiplexingStream.

Reply:
I should add a comment. I am simply testing the `MultiplexingStream` property on the formatter -- that it can retain a value and accepts a `null` assignment as well.

All the functionality we expect due to setting the property is tested in another class. I'll mention this in the comment too.